server/controllers/pages.js

```diff
@@ -16,7 +16,27 @@ const entryHelper = require('../helpers/entry')
  * Edit document in Markdown
  */
 router.get('/edit/*', (req, res, next) => {
-  if (!res.locals.rights.write) {
+  var xPaths = ['news', 'general'] // the final word in the url For.e.g : news in /edit/news, general in /edit/general 
+
+  var checkEditAvailable = function (xPaths) {
+    let paths = req.path.split('/')
+
+    let path = paths[ paths.length - 1 ]
+
+    for (let i = 0; i < xPaths.length; i++) {
+      if (path === xPaths[i]) {
```

Comment:
Change to `if(page_path.startsWith('xPaths[i])) {`.

Right now, you only seem to be comparing the very last section of the path but what of we have deeply nested things?

Reply:
Yeah but edit option would be  enabled only on the last part , right ? 
Example of what you're saying ?